City-of-Helsinki/drupal-hdbt
# UHF-X: List of links styling fixes

How to test:

1. `composer require drupal/hdbt:dev-UHF-X_list_of_links_changes`
2. `make drush-cr`
3. Go to a basic page and create three list of links paragraphs each with different design. Make sure that the font size on the links is the same on all designs and that there is bit more line-height on the links. Use the screenshot here as a visual guide.

<img width="968" alt="Screenshot 2021-09-28 at 11 33 43" src="https://user-images.githubusercontent.com/2276077/135052742-e9656d80-776b-452d-a0a2-fe35211876e8.png">
